lms/djangoapps/courseware/views.py

```diff
@@ -412,9 +438,9 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             return render_to_response('courseware/courseware.html', context)
         elif chapter is None:
             # Check first to see if we should instead redirect the user to an Entrance Exam
-            if settings.FEATURES.get('ENTRANCE_EXAMS', False) and course.entrance_exam_enabled:
-                exam_chapter, __ = get_entrance_exam_content_info(request, course)
-                if exam_chapter is not None:
+            if course_has_entrance_exam(course):
+                exam_chapter = get_entrance_exam_content(request, course)
+                if exam_chapter:
```

Comment:
Oh, anything is possible :)  There are enough moving parts when it comes to entrance exams -- modules, module metadata, milestones, toc flags, etc. -- that I don't mind the extra guarding here